src/app/collection/pages/tags/tag-with-cyber/components/header/header.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input } from '@angular/core';
+import { Router } from '@angular/router';
+import { TAGS_ROUTES } from 'app/core/learning-object-module/tags/tags.routes';
+
+interface TagsResponse {
+  tags: { _id: string }[];
+  total: number;
+}
```

Comment:
I saw this same interface was made in the stats component, we can probably throw it in the usage-stats.ts file that has the other interfaces and reference it from there.